backend/db/models/review.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+const {
+  Model
+} = require('sequelize');
+module.exports = (sequelize, DataTypes) => {
+  class Review extends Model {
+    /**
+     * Helper method for defining associations.
+     * This method is not a part of Sequelize lifecycle.
+     * The `models/index` file will call this method automatically.
+     */
+    static associate(models) {
+      // define association here
+      Review.belongsTo(models.User, { foreignKey: "userId" }),
+        Review.belongsTo(models.Spot, { foreignKey: 'spotId' }),
+        Review.hasMany(models.ReviewImage, { foreignKey: 'reviewId', onDelete: 'CASCADE' })
```

Comment:
you can also add `hooks: true` to the association object to help debug issues with cascade